### ISSUE TITLE
Unwanted extra new lines

### ISSUE THREAD

**Issue by devnoname120** [Half-Shot/matrix-appservice-discord#192]
![screenshot_2018-07-22-18-08-58](https://user-images.githubusercontent.com/2824100/43047634-9687448c-8dda-11e8-82fc-ef9b7b6e0af0.jpg)

StepS is talking from Discord, while ryojimochizuki is on Matrix.
Notice the extra new lines in StepS's messages.

Source of a message from StepS:
```json
{
  "age": 2701687,
  "content": {
    "body": "loading decrypted eboot/modules via repatch",
    "msgtype": "m.text",
    "formatted_body": "\u003cp\u003eloading decrypted eboot/modules via repatch\u003c/p\u003e\n",
    "format": "org.matrix.custom.html"
  },
  "event_id": "$153227233422767EgQHb:t2bot.io",
  "m_is_internal_pagination_token": false,
  "m_sent_state": "SENT",
  "origin_server_ts": 1532272334974,
  "room_id": "!SabTNcrymKhDfHdoaQ:xyz.is",
  "sender": "@_discord_283238983650246657:t2bot.io",
  "type": "m.room.message",
  "unsigned": {
    "age": 2701687
  },
  "user_id": "@_discord_283238983650246657:t2bot.io"
}
```

Notice the `<p>` and `\n` that shouldn't be here.

**Comment by Half-Shot** [comment]
I believe it is a symtom of using ``marked`` as our markdown engine. I guess we can manually remove the <p> tags.

**Comment by Sorunome** [comment]
That's actually a riot render bug, afaik

**Comment by Half-Shot** [comment]
It kinda is, but I also don't feel like ``<p>`` tags or newlines are exactly great things to include.

**Comment by devnoname120** [comment]
It seems to be fixed on version 0.8.17.

**Comment by Half-Shot** [comment]
Yep
